e2e/po/contact-list.po.ts

```diff
@@ -28,6 +28,17 @@ export class ContactListPageObject extends PageObject {
     this.browser.get('/');
   }
 
+  clickContactNumber(contactNumber: number) {
+    return this.trs.count().then(count => {
+      console.log(count);
+      if (contactNumber - 1 < count) {
```

Comment:
what does `contactNumber - 1` represent? is it an index?
could we do something like this to add some more context to that code:
```  javascript
const contactIndexNumber = contactNumber - 1;

if (contactIndexNumber < count) {
  return this.trs.get(contactIndexNumber).click();
}
```

Reply:
Yup. I like this suggestion because I am repeating myself with `contactNumber - 1`.